Services/Init/classes/class.ilStartUpGUI.php

```diff
@@ -1263,8 +1267,12 @@ protected function showLogout()
             $this->ctrl->setParameter($this, "client_id", "");
         }
 
+        $withdrawal_appendage_text = ilTermsOfServiceHelper::appendWithdrawalText(
+            ($DIC->http()->request()->getQueryParams()['withdrawal_relogin_content'] ?? 0)
```

Comment:
Please add an (int) cast here, because the method parameter expects an integer.

```suggestion
            (int) ($DIC->http()->request()->getQueryParams()['withdrawal_relogin_content'] ?? 0)
```